packages/app/src/cli/models/extensions/specifications/payments_app_extension_schemas/credit_card_payments_app_extension_schema.ts

```diff
@@ -26,7 +26,9 @@ export const CreditCardPaymentsAppExtensionSchema = BasePaymentsAppExtensionSche
     targeting: zod.array(zod.object({target: zod.literal(CREDIT_CARD_TARGET)})).length(1),
     verification_session_url: zod.string().url().optional(),
     ui_extension_handle: zod.string().optional(),
-    encryption_certificate_fingerprint: zod.string().optional(),
+    encryption_certificate_fingerprint: zod
+      .string()
+      .min(1, {message: "Encryption certificate fingerprint can't be blank"}),
```

Comment:
Does this require localization?

Reply:
I dont think we support localization for the CLI